### PR TITLE
Br filecoin cryptoeconnomics

### PR DESCRIPTION
**Pull request by brossetti1**
## Problem

the `Filecoin Cryptoeconomics` specifically states the following:

```
For most blockchain protocols, miners are incentivized to do work that record ledger transactions and 
do the proof of work required to move the blockchain forward. Filecoin is different in how it requires
providers to do both this, as well as provide immutable storage for the network, which is why we 
call them **Storage Providers** instead of miners.
```

the term `miner` is used around the content before and after this statement which can be more confusing than just referencing it as `storage provider`

## Solution

change instances of `miner` to `storage provider` - any reference to `miner` is secondary and referenced as `storage provider (miner)`

### CODE REVIEW

File: content/en/curriculum/filecoin/cryptoeconomics/index.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ There are several special mechanisms that the Filecoin blockchain implements, su
...
-* **Storage Provider Deal Collateral** – Miners can offer a higher deal collateral to data providers (above the minimum specified by the protocol), implying a higher level of service and reliability. Given the increased stakes, clients may associate collateral beyond the minimum with an increased likelihood that their data will be reliably stored. This is returned to storage providers once sectors expire.
+* **Block Reward as Collateral** – Filecoin penalizes Storage Providers that fail to store files for the promised duration, balancing it with an overly high cost to join the network. Block rewards earned by a Storage Providers are slashed if a sector is terminated before its expiration.
+* **Storage Provider Deal Collateral** – Storage Providers can offer a higher deal collateral to data providers (above the minimum specified by the protocol), implying a higher level of service and reliability. Given the increased stakes, clients may associate collateral beyond the minimum with an increased likelihood that their data will be reliably stored. This is returned to storage providers once sectors expire.
```
Comment (brossetti1) [inline content/en/curriculum/filecoin/cryptoeconomics/index.md:53]: @walkerlj0 if this is acceptable, im wondering if `Storage Provider(s)` should be capitalized? or lowercase?

**Comment by brossetti1** [comment]
would you like me to do this conversion everywhere in the launchpad program?

File: content/en/curriculum/filecoin/cryptoeconomics/index.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ In this 6 minute video, understand how Filecoin proves both replication, retriev
 
 ### [Storage Providers (Miners)](https://docs.filecoin.io/storage-provider/how-providing-works)
 
-* Anyone with the [minimum required hardware](https://docs.filecoin.io/storage-provider/hardware-requirements/) can create a storage miner
-* Miners do not earn rewards until they onboard a minimum amount of sealed storage to the network
+* Anyone with the [minimum required hardware](https://docs.filecoin.io/storage-provider/hardware-requirements/) can create a storage Storage Provider
```
Comment (walkerlj0) [inline content/en/curriculum/filecoin/cryptoeconomics/index.md:41]:
```suggestion
* Anyone with the [minimum required hardware](https://docs.filecoin.io/storage-provider/hardware-requirements/) can be a storage Storage Provider
```

**Review by walkerlj0 — APPROVED** [review]
One minor typo fix to add on, awesome thank you!

**Pull request merged**